src/components/Dashboard/index.js

```diff
@@ -8,27 +29,18 @@ import {
   clearProfileEditError,
   setUpInitialData,
 } from "../../store/actions";
-import {
-  GlobalOutlined,
-  UserAddOutlined,
-  AppstoreAddOutlined,
-  AppstoreOutlined,
-  IeOutlined,
-} from "@ant-design/icons";
+
 import countryList from "../../helpers/countryList";
 import orgUser from "../../assets/images/org-user.svg";
 import profileUser from "../../assets/images/profile-user.svg";
 import Fade from "react-reveal/Fade";
-import {
-  orgWebsiteValidation,
-  orgHandleValidation,
-  userHandleValidation,
-} from "../../helpers/validationRules";
 
-const { Option } = Select;
+import { validateName } from "../../helpers/validations";
+import { validateHandle } from "../../helpers/validations";
+import { validateCountry } from "../../helpers/validations";
+import { validateOrgWebsite } from "../../helpers/validations";
```

Comment:
I don't think this importing method is quite correct in this case. Need approval from @Niweera.

Reply:
Yes you can combine all the import in a single row. Please refer to the ES6 imports.

Reply:
As we are importing Material UI components separately for this project, I thought it will be better if I go with this. Sorry. I will get them to a single line.

Reply:
Combined and got them to a single line. Thank you @shehand and @Niweera